kits/automation/blog-automation/.env.example

```
@@ -0,0 +1,9 @@
+# Lamatic Flows
+AUTOMATION_BLOG_DRAFTING = "AGENTIC_GENERATE_CONTENT FLOW ID"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)